pages/windows/ospp.vbs.md

```diff
@@ -0,0 +1,29 @@
+# ospp.vbs
+
+> Install, activate, and manage volume licensed versions of Office products.
+> This command may override, deactivate, and/or remove your current volume licensed versions of Office products. Please proceed with caution.
+> More information: <https://learn.microsoft.com/deployoffice/vlactivation/tools-to-manage-volume-activation-of-office>.
+
+- Install a product key (replace existing key):
+
+`cscript ospp.vbs /inpkey:{{product_key}}`
+
+- Uninstall an installed product key with the last five digits of the product key:
+
+`cscript ospp.vbs /unpkey:{{last_five_digits_of_the_product_key}}`
```

Comment:
```suggestion
`cscript ospp.vbs /unpkey:{{product_key}}`
```
Since it's already specified in the description, I don't think we need to repeat this again.